_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+
+In Bitcoin and other cryptocurrencies, scalable commitments to sets of
+information---such as transactions or UTXOs---are normally made using
+merkle trees that allow proving an element is a member of the set by
+generating a proof whose size and validation cost is roughly *log2(n)*
+for a set of *n* elements.
+
+This talk describes an alternative method based on RSA accumulators that
+provides many potential benefits: the size of a proof is constant no
+matter how many elements are members of the set, adding or removing
+elements from an accumulator can be efficiently batched (e.g. one update
+per block), and validation that one or more elements are members of a
+set could potentially be much faster (although this has not yet been
+fully tested).
+
+*[video][vid accumulators], [transcript][tx accumulators], soft fork for
+new applications such as UTXO set commitments*
+
+### Scriptless ECDSA
+
+*Research by Conner Fromknecht*
+
+At Scaling Bitcoin IV (Stanford 2017), Andrew Poelstra
+[presented][scriptless scripts transcript] his existing "scriptless
+scripts" work that suggests several ways outputs can be encumbered in
+similar ways to a few of the things now possible with Bitcoin Script but
+by using only Schnorr-based keys and signatures.  This method has the
+advantage of being just as secure as the Script-based mechanism but also
+maximally compact, faster to validate, and completely invisible to third
+parties (providing privacy, increased fungibility, and potentially
+increased security from the fungibility).
+
+Bitcoin doesn't currently support Schnorr signatures and no complete
+design for it has been proposed (although such a proposal may not be far
+off), so this talk describes proof-of-concept results from a partial
+implementation of scriptless-scripts that's compatible with Bitcoin's
+current keys and signatures, ECDSA.  The talk especially focuses on the
+use of scriptless scripts in payment channels such as those used by
+Lightning Network, with some impressive savings in the size of scripts
+and witness data---savings which increase the number of channels that
+can be opened or closed in a block and which reduces the amount of
+transaction fee paid by users of Lightning Network payment channels.
+
+*[video][vid scriptless ecdsa], [transcript][tx scriptless ecdsa], possible
+now*
+
+## Discussion wrap-up: the evolution of bitcoin script
+
+*Discussion wrap-up presented by Olaoluwa Osuntokun*
+
+This wrap-up of a two-hour discussion group briefly mentioned a large
+variety of proposed changes to Bitcoin's Script language---far too many
+to mention here even in summary.  However, a few changes were mentioned
+as theoretically possible to accomplish in 2019 if the community is
+willing to adopt them:
+
+- **Schnorr signature scheme:** an opt-in feature providing smaller
+  signatures in all cases, much smaller public key and signature data
+  for cooperative multisigs, and faster validation.  Easier
```

Comment:
'Easier compatibility with scriptless scripts' is an incomplete sentence. Perhaps squash with previous sentence?